components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -302,6 +351,73 @@ class NewTabPage extends React.Component<Props, State> {
     window.open('https://brave.com/binance/', '_blank')
```

Comment:
set `noopener`(technically this isn't part of this PR but i was never tagged for review on the original one)